src/main.ts

```diff
@@ -169,6 +169,8 @@ interface DiceRollerSettings {
     initialDisplay: ExpectedValue;
 
     icons: DiceIcon[];
+    diceModTemplateFolder: string;
```

Comment:
I think this would be better as an ignore list, vs a single template folder.

Reply:
Well it's the folder and all its subfolders, but I'll.look up how to make it a list